internal/backends/qt/qt_window.rs

```diff
@@ -1714,6 +1714,37 @@ impl PlatformWindow for QtWindow {
     fn as_any(&self) -> &dyn std::any::Any {
         self
     }
+
+    fn handle_focus_change(&self, old: Option<ItemRc>, new: Option<ItemRc>) {
+        println!("QtWindow: Focus changed from {old:?} to {new:?}.");
```

Comment:
Leftover debug

Reply:
I think that is removed again in a later patch.